orga/tests/integration/components/auth/login-form_test.js

```diff
@@ -140,13 +140,37 @@ module('Integration | Component | Auth::LoginForm', function (hooks) {
 
     // then
     assert.dom('#login-form-error-message').exists();
+    const expectedErrorMessages = this.intl.t('api-error-messages.login-unauthorized-error');
+    assert.dom('#login-form-error-message').hasText(expectedErrorMessages);
+  });
+
+  test('it displays a should change password message', async function (assert) {
+    // given
+    const errorResponse = {
+      responseJSON: {
+        errors: [{ status: '401', code: 'SHOULD_CHANGE_PASSWORD' }],
+      },
+    };
+
+    SessionStub.prototype.authenticate = () => reject(errorResponse);
+
+    await renderScreen(hbs`<Auth::LoginForm/>`);
+    await fillByLabel(emailInputLabel, 'pix@example.net');
+    await fillByLabel(passwordInputLabel, 'Mauvais mot de passe');
+
+    //  when
+    await clickByName(loginLabel);
+
+    // then
+    assert.dom('#login-form-error-message').exists();
+    const expectedErrorMessages = this.intl.t('pages.login-form.errors.status.401-should-change-password');
     assert.dom('#login-form-error-message').hasText(expectedErrorMessages);
```

Comment:
```suggestion
assert.dom(screen.getByText(this.intl.t('pages.login-form.errors.status.401-should-change-password');)).exists();
```
Je me doute que c'est un copié collé des exemples d'à côté mais c'est une mauvaise pratique de selectionner une balise par un élément technique (ici l'id). Le mieux est de rester plus souple en faisant une assertion qui dit juste "vérifie que mon texte existe dans la page".

Reply:
Modification faite suivant la recommandation.